Quorum/ipfs_validator.py

```diff
@@ -0,0 +1,69 @@
+from Quorum.utils.chain_enum import Chain
+import Quorum.config as config
+from Quorum.apis.block_explorers.chains_api import ChainAPI
+from Quorum.apis.llms.claude_conversation import ClaudeConversation
+
+import requests
+import argparse
+import re
+from pathlib import Path
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description='Compare ipfs with actual payload.')
+    parser.add_argument('--proposal_id', type=int, help='The id of the proposal.')
+    parser.add_argument('--chain', type=str, choices=[chain.value for chain in Chain], help='Blockchain chain.')
+    parser.add_argument('--proposal_address', type=str, help='Ethereum proposal address.')
+    parser.add_argument('--prompt_template', type=Path, default=Path(__file__).parent / 'ipfs_validation_prompt.txt',
+                        help='Optional other template for prompting the LLM.')
+    
+    args = parser.parse_args()
+    if not args.prompt_template.exists():
+        raise argparse.ArgumentError(f'Could not find prompts at {args.prompt_template}.')
+
+    return args
+
+
+def get_raw_ipfs(proposal_id: int) -> str:
```

Comment:
Personally, I dont like having all this API calls without a handler.
Dont we have already handler for this communication?

Reply:
Not really. We do have something a bit more organized for the automatic report PR, but I didn't want these to be coupled. Also this whole function is temporary, I should add this as a comment.